guides/7-jlp/4-JLP-Economics.md

```diff
@@ -0,0 +1,142 @@
+* USDT: [4vkNeXiYEUizLdrpdPS1eC2mccyM4NUPRtERrk6ZETkk](https://solscan.io/account/4vkNeXiYEUizLdrpdPS1eC2mccyM4NUPRtERrk6ZETkk)
+
+More info on the `Custody` account is explained in [Onchain Account Types](../8-perpetual-exchange/3-onchain-accounts.md).
+
+### Assets Under Management
+
+The AUM for each `Custody` account in the pool is calculated as follows:
+
+#### Stablecoins (USDC, USDT)
+
+`aum = owned * current_price`
+
+#### Non-stablecoins (SOL, ETH, wBTC)
+
+1. Calculate the global short position profit/loss (Unrealized PnL in USD):
+
+`unrealized_pnl = (global_short_sizes * (|global_short_average_prices - current_price|)) / global_short_average_prices)`
+
+:::tip
+If `current_price` > `global_short_average_prices`, traders are losing on short positions.
+:::
+
+2. Net Asset Value (NAV):
+
+`available_tokens = owned - locked`
+
+`nav = available_tokens * current_token_price`
+
+`nav += guaranteed_usd`
+
+:::info
+The `guaranteed_usd` value in each `Custody` account represents an estimate of the total size of all long positions. It is only an estimate as `guaranteed_usd` is only updated when positions are updated (i.e. opening / closing positions, updating collateral). It does not update in real-time when asset prices change. 
+
+`guaranteed_usd` is used to calculate the pool's AUM as well as the overall PnL for all long positions efficiently.
+:::
+
+3. Assets Under Management (AUM)
+
+If traders are losing on short positions, the losses are added to the pool's AUM:
+
+`aum = nav + unrealized_pnl`
+
+Otherwise, traders' profits are deducted from the pool's AUM:
+
+`aum = nav - unrealized_pnl`
+
+The Total AUM is then calculated as the sum of all the custodies' AUM:
+
+`total_aum = Σ(aum)`
+
+:::note
+The pool sizes displayed on the [Earn page](https://jup.ag/perps-earn) do not add up to the pool's AUM. This discrepancy occurs because the Earn page shows each custody's `owned` value in USD without accounting for:
+
+1. Traders' unrealized PnL
+2. The `locked` amount reserved for potential profit payouts on open positions
+
+This simplified representation on the Earn page provides a quick overview of the pool's holdings but doesn't reflect the JLP pool's true AUM.
+:::
+
+### Virtual Price, Market Price and AUM Limit
+
+`Virtual Price = Sum of all JLP Assets (in USD) / Total Quantity of JLP in circulation`
+
+When AUM limit is hit:
+
+`Market Price = Virtual Price + Market-assigned Premium`
+
+Usually, users can mint new JLP or redeem (burn) them at the Virtual Price. However, when AUM Limits are hit, new minting of JLP is disabled to cap the amount of TVL in the pool.
+
+When this happens, the demand for JLP on the market usually leads to a premium for JLP compared to the virtual price.
+
+You may sell your JLP for the Market Price at any time. If the Market Price is below the Virtual Price, your JLP tokens are redeemed (burned) at the virtual price instead of sold at the market price.
```

Comment:
```suggestion
You may sell your JLP for the Market Price at any time. If the Market Price is below the Virtual Price, your JLP tokens are redeemed (burned) at the virtual price instead of the market price.
```